garak/resources/gcg/generate_gcg.py

```diff
@@ -94,7 +94,7 @@ def run_gcg(
     / "resources/gcg/data/advbench/harmful_behaviors.csv",
     n_train: int = 50,
     n_test: int = 0,
-    outfile: str = garak._config.transient.basedir / "resources/data/gcg/gcg.txt",
+    outfile: str = garak._config.transient.basedir / "resources/gcg/data/gcg/gcg.txt",
```

Comment:
I guess there's an agreement between the probes/ code and the code in resources/ that this will be the location. Maybe this could (one day) come from _config but that needn't block this PR

Reply:
That's a good idea, tbh. If you have your own stuff you want to jam in configs. I think we need a separate PR where we adjust `_config` -- may be a good place to deal with some of the model parameter stuff too.